test/allocators/LUSDAllocator.test.ts

```diff
@@ -2,229 +2,537 @@ import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signers";
-        await lusdAllocator.connect(owner).withdraw(lusdTokenFake.address, PARTIAL_AMOUNT);
+        ;
 
-        expect(stabilityPoolFake.withdrawFromSP).to.be.calledWith(PARTIAL_AMOUNT);
-        expect(treasuryFake.deposit).to.be.calledWith(PARTIAL_AMOUNT, lusdTokenFake.address, PARTIAL_VALUE);
-        expect(lusdTokenFake.balanceOf).to.be.calledWith(lusdAllocator.address);
-        expect(lusdTokenFake.approve).to.be.calledWith(treasuryFake.address, PARTIAL_AMOUNT);
+      await advance(100);
 
-        expect(await lusdAllocator.totalAmountDeployed()).to.equal(DEPOSIT_AMOUNT - PARTIAL_AMOUNT);
-        expect(await lusdAllocator.totalValueDeployed()).to.equal(DEPOSIT_VALUE - PARTIAL_VALUE);
-      });
+      ethRewards = await allocator.getETHRewards();
+      lqtyRewards = await allocator.getLQTYRewards();
 
-      it("reverts if non-LUSD token is passed", async () => {
-        await expect(lusdAllocator.connect(owner).withdraw(other.address, 12345))
-          .to.be.revertedWith("token address does not match LUSD token");
-      });
+      expect(ethRewards).to.equal(ZERO);
+      expect(lqtyRewards).to.equal(ZERO);
 
-      it("can only be called by the owner", async () => {
-        await expect(lusdAllocator.connect(other).withdraw(lusdTokenFake.address, 12345))
-          .to.be.revertedWith("Ownable: caller is not the owner");
-      });
+        // .to.emit(lusdStabilityPool, "G_Updated").withArgs(ethers.BigNumber.from("350461943063989161432445055694169347038"), 0, 0)
+        // // .not.to.emit(lusdStabilityPool, "LQTYPaidToFrontEnd").withArgs(ZERO_ADDRESS, 0)          //How to get NOT emit to work??!
+        // .to.emit(lusdStabilityPool, "LQTYPaidToDepositor").withArgs(allocator.address, 0)
+        // .to.emit(lusdStabilityPool, "FrontEndStakeChanged").withArgs(ZERO_ADDRESS, ethers.BigNumber.from("45632567909241122202977270"), allocator.address)
+        // .to.emit(lusdStabilityPool, "StabilityPoolLUSDBalanceUpdated").withArgs(STABILITY_POOL_TOTAL_BALANCE)
+        // .to.emit(lusdStabilityPool, "DepositSnapshotUpdated").withArgs(allocator.address,
+        //   ethers.BigNumber.from("876920926160447076"), ethers.BigNumber.from("58089263752322121983911170457988"), ethers.BigNumber.from("350461943063989161432445055694169347038"))
+        // .to.emit(lusdStabilityPool, "UserDepositChanged").withArgs(allocator.address, DEPOSIT_AMOUNT)
+        // .to.emit(lusdStabilityPool, "ETHGainWithdrawn").withArgs(allocator.address, ZERO, ZERO)
+        // .to.emit(oldTreasury, "ReservesUpdated").withArgs(ethers.BigNumber.from("168613073893284949"))
+        // .to.emit(oldTreasury, "ReservesManaged").withArgs(LUSD_TOKEN_ADDRESS, DEPOSIT_AMOUNT)
+        // ;
+
+      await allocator.connect(manager).withdraw(LUSD_TOKEN_ADDRESS, DEPOSIT_AMOUNT);
+        // .to.emit(lusdStabilityPool, "G_Updated").withArgs(ethers.BigNumber.from("350461943753146787543868841926858937250"), 0, 0)
```

Comment:
Assuming a WIP?

Reply:
removed the commented out lines.  was trying to do some crazy tests that didn't work